.github/workflows/publish-extensions.yml

```diff
@@ -43,19 +54,28 @@ jobs:
       - name: Sign Production Firefox version
         continue-on-error: true
         run: yarn web-ext sign --channel=unlisted
```

Comment:
Will update to `listed` before merging.

Reply:
Let's go ahead and make that change, and then merge this. Right now trying to manually publish doesn't really work without a manual version bump (locally), because there is a conflicting version in the `unlisted` channel